### ISSUE TITLE
Submission UI should reflect if a VPN is set as the default route

### ISSUE THREAD

**Issue by seanthegeek** [cuckoosandbox/cuckoo#1655]
Currently, if a VPN is set as the default route, it will work fine, but the task submission page should show that prior to submission.

**Comment by SparkyNZL** [comment]
My understanding is that you can already set that in the .conf file, my
default is TOR and not VPN or Internet

From memory it's in the Routing.conf file, just off the top of my head

Cheers

**Comment by seanthegeek** [comment]
Right, you set it in the config file, and the default VPN does work just like Tor would. However, the VPN option doesn't show up as selected by default in the UI, even though it actually is. The other routing options don't have this problem because they don't use a dropdown.
